doc/wg/core/notes/core-notes-2020-04-17.md

```diff
@@ -0,0 +1,98 @@
+# Tock Core Notes 4/17/2020
+
+Attending:
+ * Branden Ghena
+ * Amit Levy
+ * Brad Campbell
+ * Vadim Sukhomlinov
+ * Johnathan Van Why
+ * Pat Pannuto
+ * Samuel Jero
+ * Alistair
+ * Garret Kelly
+ * Philip Levis
+ * Andrey Pronin
+ * Hudson Ayers
```

Comment:
I did join fairly late; it might create confusion to see my comments while not having attended. :)

Reply:
Fixed!  Sorry about that.

Reply:
Thanks and no worries!